src/modules/users/services/__mock__/user.mock.ts

```diff
@@ -9,6 +10,7 @@ export const mockUserResponse: { users_profile: GqlUser[] } = {
 			last_name: 'Odhiambo',
 			full_name: 'Marie Odhiambo',
 			mail: 'marie.odhiambo@example.com',
+			language: Lookup_Languages_Enum.Nl,
```

Comment:
prefer the alias Locale instead of the underlying database type: Lookup_Languages_Enum
```suggestion
			language: Locale.Nl,
```

Reply:
The user profile expects the type Lookup_Languages_Enum and will error on Locale.nl